### ISSUE TITLE
globo.com: breakage

### ISSUE THREAD

**Issue by KeeperOfScrolls** [uBlockOrigin/uAssets#17368]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Videos are broken on globo.com.

globo.com is the largest newspaper in Brazil (200M people), so it's kinda of an annoyance.

Related issues:
- https://github.com/uBlockOrigin/uAssets/issues/17332
- https://github.com/uBlockOrigin/uAssets/issues/16369
- https://github.com/uBlockOrigin/uAssets/issues/14379

This filter solves it:

```
! 2023-04-01 https://globo.com
@@||*/chartbeat/*$script,domain=globo.com
```

The issue seems to be that EasyPrivacy line 1036 will block this script:

```
/chartbeat/*
```

This blocks chartbeat.js, which probably causes some kind of JS error that prevents videos from working.

### URL(s) where the issue occurs.
**Search for a video on the page.**
```
- https://g1.globo.com/rj/rio-de-janeiro/noticia/2023/03/31/defesa-de-bruno-krupp-entrega-passaporte-do-modelo-e-pede-sigilo-para-seu-endereco.ghtml

- https://oglobo.globo.com/mundo/noticia/2023/03/brasileira-registra-passagem-de-tornado-nivel-5-no-estado-americano-do-arkansas-veja-video.ghtml 
```
(You might need Burlesco to view this one)

### Screenshot(s)

![image](https://user-images.githubusercontent.com/122753641/229259016-f41a5fb0-6d90-448e-81a1-c29dd120c851.png)

### uBO version

1.48.0

### Browser name and version

Brave Browser 1.49

### Settings

Added the filter below to solve my problem:
```
! 2023-04-01 https://globo.com
@@||*/chartbeat/*$script,domain=globo.com
```
I'm opening this here to help others.

**Comment by mapx-** [comment]
The video is playing fine (in chrome) without no exception on my side.
Did you disable brave shields for this site ?

**Comment by mapx-** [comment]
tested in brave too, still no issue on my end

**Comment by ryanbr** [comment]
which lists enabled in brave://adblock ? @KeeperOfScrolls

**Comment by mapx-** [comment]
@KeeperOfScrolls force update all filter lists, see if works now

**Comment by KeeperOfScrolls** [comment]
I find it intriguing that you cannot replicate this issue. For my tests, I have disabled Brave Shields and only have Ublock Origin active. Moreover, three related issues have reported the same problem.

> which lists enabled in brave://adblock ?

I have the following lists enabled:

- Easylist - Cookie List - Filter Obtrusive Cookie Notices
- Fanboy's Mobile Notifications List

As previously mentioned, from my tests, the problem seems to come from EasyPrivacy line 1036: `/chartbeat/*`

I have documented the issue in a video, I hope it helps:
https://youtu.be/eb8XW06Vt0U

**Comment by KeeperOfScrolls** [comment]
> force update all filter lists, see if works now

Thanks for the suggestion. I've purged all caches and force updated, got the same results.

**Comment by mapx-** [comment]
I already "killed" that filter (in unbreak list)

`/chartbeat/*$badfilter`

see if you still see it in the logger or force update again the lists
or maybe it's some other filter breaking the page on your side

**Comment by KeeperOfScrolls** [comment]
> chartbeat

Indeed, I see your rule when I go to uBlock Origins -> Filters List -> uBlock filters - Unbreak -> View content:

```
! disable too broad filter
/chartbeat/*$badfilter
```

Ah, I see now. The filter that is causing the issue is actually `/chartbeat-`:

![image](https://user-images.githubusercontent.com/122753641/229316003-2d4e08b6-85c2-478e-a666-957d547fb565.png)

So your "unbreak filter" wont match that one because it ends in `-`, not `/`. Is there anything we can do?

**Comment by KeeperOfScrolls** [comment]
Maybe allow it only on `*.globo.com`, if possible?

**Comment by mapx-** [comment]
Are you sure ?

add it and test again

`@@||glbimg.com/*/plugins/chartbeat/chartbeat-plugin.min.js$script,domain=globo.com`

**Comment by KeeperOfScrolls** [comment]
> Are you sure ?
> 
> add it and test again
> 
> `@@||glbimg.com/*/plugins/chartbeat/chartbeat-plugin.min.js$script,domain=globo.com`

I can assert that without any custom filters, the videos in the URLs under test are broken.

When I add this filter that you just shared, the videos works. In the Logger, I see:

![image](https://user-images.githubusercontent.com/122753641/229319381-444ac28c-7b98-4622-8e81-96d829aebbb4.png)

The behavior is reproducible for me with/without the filter. I'm pinging the users that also reported this issue, @hiuahiuahiua, @jakecharlie (Vocês podem ajudar a verificar se esse filtro que o mapx passou resolve o bug do vídeo pra vocês?)

**Comment by KeeperOfScrolls** [comment]
@jakecharlie The faulty filter that is currently cause an issue is from the EasyPrivacy list as well, see https://github.com/uBlockOrigin/uAssets/issues/17368#issuecomment-1493137801. My list is up-to-date (`Sat, 01 Apr 2023 13:07:37 +0000`).

> ublock origin can do nothing about this.

Actually they can, they have a list of overrides in case a third party filter list causes issues.

There are 2 URLs in the main comment of this issue, do the videos play normally for you? If so, can you open uBlock Origin Logger, search for `chartbeat` and report back whether it's blocked or allowed?

IMO, we should just allow `chartbeat-` in globo.com and call it a day.

**Comment by KeeperOfScrolls** [comment]
> Click at the top on "Purge all caches" to delete the cache of all lists and then on "update now". Maybe it's the solution.

I have already done that.

One thing is catching my eye, though.
- My `EasyList` has 65,057 used out of 65,103
- Your `EasyList` has 42,339 used out of 65,103
- My `EasyPrivacy` has 29,580 used out of 30,755
- Your `EasyPrivacy` has 14,789 used out of 30,755

I don't know where this difference comes from, but perhaps the faulty filter isn't loaded on your end?

![image](https://user-images.githubusercontent.com/122753641/229323831-3e3bb5ba-4273-44ee-ba01-52f70d164af4.png)

**Comment by KeeperOfScrolls** [comment]
It seems this comes from my own extensions. I'll be closing this issue.

When I searched for "globo.com" on this repo and found 3 other issues describing exactly the same error as me, and when I could reproduce and isolate the behavior to the rules `/chartbeat/*` and then to `/chartbeat-` _(as removing the first triggered the second)_, I strongly believed this to be a reproducible error for everyone.

When y'all said you couldn't reproduce it, I didn't understand why. But indeed, I have installed Firefox and Chrome, tried to reproduce it here and couldn't.

So focusing on how this affects only Brave, I have tried it on a new anonymous window with just uBlock Origin and one other extension active, and I couldn`t reproduce it anymore. So this comes from my own extensions.

I think that when the `chartbeat` script is missing, it probably causes a JavaScript error that halts execution and leaves the videos in an unplayable state.

So I'll be closing this as non-reproduceable. If this pops up again for someone else we can debug it further. If it happens again I'll hop-in in here again.

Thanks, and sorry.
